lms/djangoapps/grades/config/waffle.py

```diff
@@ -2,7 +2,7 @@
 This module contains various configuration settings via
 waffle switches for the Grades app.
 """
-from openedx.core.djangolib.waffle_utils import WaffleSwitchPlus
+from openedx.core.djangoapps.waffle_utils import WaffleSwitchPlus
```

Comment:
I'll have to review the code.  As @nasthagiri mentioned, it may make sense to have waffle_utils, which may go back to being a plain old python utility (not a django app).  Then, this feature_flags django app would make use of waffle (as well as other things as you mentioned).